python-checks/src/test/resources/checks/identityComparisonWithCachedTypes.py

```diff
@@ -18,6 +18,9 @@ def literal_comparison(param):
     (4, 5) is param # Noncompliant {{Replace this "is" operator with "=="; identity operator is not reliable here.}}
     #      ^^
 
+    param is -1 #Noncompliant
```

Comment:
_Nitpick:_ We usually add a space `# Noncompliant`.
Also, just out of curiosity did you add these cases because of regressions on ruling?

Reply:
Yes, it was from a ruling